doc/user/DESKTOP_INTEGRATION.md

```diff
@@ -21,7 +21,19 @@ sudo update-mime-database ~/.local/share/mime/
 sudo update-desktop-database ~/.local/share/applications
 ```
 
-If you have any issues, refer to the [troubleshooting](LIMITATIONS_AND_TROUBLESHOOTING.md) part.
+F3D supports rendering in environments with limited graphical environnements, including sandboxes or without Xorg. In most cases, the default behavior will automatically detect which capabilities are available and use the right rendering backend, it is possible to specify it using `--rendering-backend` CLI option.
```

Comment:
Avoid repetition and specify "running" for Xorg
```suggestion
F3D supports rendering in environments with limited graphical contexts, including sandboxes or without Xorg running. In most cases, the default behavior will automatically detect which capabilities are available and use the right rendering backend, it is possible to specify it using `--rendering-backend` CLI option.
```

Reply:
you can take @Meakk suggestion into account, either by modifying as suggested or clicking on "commit suggestion".